ecosystem/typescript/sdk_v2/src/types/index.ts

```diff
@@ -155,6 +155,10 @@ export type PendingTransactionResponse = {
   signature?: TransactionSignature;
 };
 
+export function isPendingTransaction(txn: TransactionResponse): txn is PendingTransactionResponse {
```

Comment:
I dont think this function should go here, maybe in a utils/helpers file

Reply:
Using 'Discriminating Unions' instead